internal/lpastore/lpa.go

```diff
@@ -271,6 +271,7 @@ type lpaResponse struct {
 
 func lpaResponseToLpa(l lpaResponse) *lpadata.Lpa {
 	data := &lpadata.Lpa{
+		InStore:      true,
```

Comment:
I wanted to avoid using an inference on other fields and just give us a simple boolean to use